doc/source/whatsnew/v1.4.0.rst

```diff
@@ -664,6 +664,7 @@ I/O
 - Bug in :func:`read_csv` raising ``AttributeError`` when attempting to read a .csv file and infer index column dtype from an nullable integer type (:issue:`44079`)
 - :meth:`DataFrame.to_csv` and :meth:`Series.to_csv` with ``compression`` set to ``'zip'`` no longer create a zip file containing a file ending with ".zip". Instead, they try to infer the inner file name more smartly. (:issue:`39465`)
 - Bug in :func:`read_csv` when passing simultaneously a parser in ``date_parser`` and ``parse_dates=False``, the parsing was still called (:issue:`44366`)
+- Bug in :func:`read_json` raising ``ValueError`` when attempting to parse json strings containing urls (:issue:`36271`)
```

Comment:
Were all urls causing issues or only s3? otherwise lgtm

Reply:
All URLs, actually a more accurate description would be `... to parse json strings containing "://"`